packages/@react-spectrum/s2/style/properties.json

```
@@ -166,7 +166,8 @@
     "touchAction": "__X",
     "translate": "__Y",
     "wordBreak": "__Z",
-    "--fs": "__0"
+    "overflowWrap": "__0",
+    "--fs": "__1"
```

```suggestion
    "--fs": "__0",
    "overflowWrap": "__1"
```

@devongovett i think this has to be like this to keep it stable across versions?

---

Should we just swap the value as you suggested @snowystinger to be safe, and move on?

---

Oh cool, so we can just remove all the changes made to this file?